kornia/augmentation/container/augment.py

```diff
@@ -107,129 +129,14 @@ def __init__(
-        if DataKey.get(dcate) in [DataKey.BBOX_XYHW]:
-            if isinstance(input, (tuple,)):
-                return (self.apply_to_bbox(input[0], module, param, mode='xyhw'), *input[1:]), None
-            return self.apply_to_bbox(input, module, param, mode='xyhw'), None
-        if DataKey.get(dcate) in [DataKey.KEYPOINTS]:
-            if isinstance(input, (tuple,)):
-                return (self.apply_to_keypoints(input[0], module, param), *input[1:]), None
-            return self.apply_to_keypoints(input, module, param), None
-        raise NotImplementedError(f"input type of {dcate} is not implemented.")
-
-    def inverse_input(self, input: torch.Tensor, module: nn.Module, param: Optional[ParamItem] = None) -> torch.Tensor:
-        if isinstance(module, GeometricAugmentationBase2D):
-            input = module.inverse(input, None if param is None else cast(Dict, param.data))
-        return input
-
-    def inverse_bbox(
-        self, input: torch.Tensor, module: nn.Module, param: Optional[ParamItem] = None, mode: str = "xyxy"
-    ) -> torch.Tensor:
-        if isinstance(module, GeometricAugmentationBase2D):
-            transform = module.compute_inverse_transformation(
-                module.get_transformation_matrix(input, None if param is None else cast(Dict, param.data))
-            )
-            input = transform_bbox(torch.as_tensor(transform, device=input.device, dtype=input.dtype), input, mode)
-        return input
-
-    def inverse_keypoints(
-        self, input: torch.Tensor, module: nn.Module, param: Optional[ParamItem] = None
-    ) -> torch.Tensor:
-        if isinstance(module, GeometricAugmentationBase2D):
-            transform = module.compute_inverse_transformation(
-                module.get_transformation_matrix(input, None if param is None else cast(Dict, param.data))
-            )
-            input = transform_points(torch.as_tensor(transform, device=input.device, dtype=input.dtype), input)
-        return input
-
-    def inverse_by_key(
-        self,
-        input: torch.Tensor,
-        module: nn.Module,
-        param: Optional[ParamItem] = None,
-        dcate: Union[str, int, DataKey] = DataKey.INPUT,
-    ) -> torch.Tensor:
-        if DataKey.get(dcate) in [DataKey.INPUT, DataKey.MASK]:
-            return self.inverse_input(input, module, param)
-        if DataKey.get(dcate) in [DataKey.BBOX, DataKey.BBOX_XYXY]:
-            return self.inverse_bbox(input, module, param, mode='xyxy')
-        if DataKey.get(dcate) in [DataKey.BBOX_XYHW]:
-            return self.inverse_bbox(input, module, param, mode='xyhw')
-        if DataKey.get(dcate) in [DataKey.KEYPOINTS]:
-            return self.inverse_keypoints(input, module, param)
-        raise NotImplementedError(f"input type of {dcate} is not implemented.")
-
-    def inverse(
+    def inverse(  # type: ignore
         self,
         *args: torch.Tensor,
         params: Optional[List[ParamItem]] = None,
```

Comment:
Mismatched parameters in overridden method
https://deepsource.io/gh/kornia/kornia/run/3492ca94-6546-4722-8a23-f570db6834d1/python/PYL-W0221

@shijianjian I saw that error before during code cleanup. Is there any way we can force or test that ?

Reply:
I think it is an error that we shall ignore. It happens all the time when we override a ```forward``` method with params more than one tensor. Similar here and that's why we used ```type:ignore``` to it. Probably we should make Deepsource ignore it too.